packages/front-end/components/Settings/UpgradeModal/SelfHostedUpgradeForm.tsx

```diff
@@ -83,10 +146,10 @@ export default function SelfHostedUpgradeForm({
             <a
               href="mailto:sales@growthbook.io"
               onClick={() => {
-                track("Click Cloud custom quote email", {
-                  accountPlan,
-                  source,
-                });
+                track(
+                  "Self hosted: Click Cloud custom quote email",
+                  customerContext
+                );
```

Comment:
Same here re tracking context.  Also, for the event name, I would match what we use in the cloud modal: `Click Enterprise Upgrade Link`